lib/_http_server.js

```diff
@@ -281,7 +273,7 @@ function connectionListener(socket) {
   if (self.timeout)
     socket.setTimeout(self.timeout);
   socket.on('timeout', function() {
-    var req = socket.parser && socket.parser.incoming;
+    var req = parser && parser.incoming;
```

Comment:
@chrisdickinson should I deal with [this](https://github.com/iojs/io.js/blob/v1.x/lib/_http_server.js#L314) and drop them all beneath?